From eaa9168ac9ca1555742e2037e82d0cda67418985 Mon Sep 17 00:00:00 2001
From: Robby Cai <R63905@freescale.com>
Date: Fri, 28 Sep 2012 17:25:25 +0800
Subject: ENGR00225981-2 pxp: fix the coefficient for the YCbCr-RGB CSC

Fix the coefficient for the YCbCr-RGB Color space conversion.

Signed-off-by: Robby Cai <R63905@freescale.com>
---
 drivers/dma/pxp/pxp_dma_v2.c |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/drivers/dma/pxp/pxp_dma_v2.c b/drivers/dma/pxp/pxp_dma_v2.c
index 90e675d..b74f62ca 100644
--- a/drivers/dma/pxp/pxp_dma_v2.c
+++ b/drivers/dma/pxp/pxp_dma_v2.c
@@ -657,7 +657,7 @@ static void pxp_set_csc(struct pxps *pxp)
 
 		/* CSC1 - YUV->RGB */
 		__raw_writel(0x84ab01f0, pxp->base + HW_PXP_CSC1_COEF0);
-		__raw_writel(0x01230204, pxp->base + HW_PXP_CSC1_COEF1);
+		__raw_writel(0x01980204, pxp->base + HW_PXP_CSC1_COEF1);
 		__raw_writel(0x0730079c, pxp->base + HW_PXP_CSC1_COEF2);
 
 		/* CSC2 - Bypass */
-- 
1.7.9.5

